testutils/clustermanager/util.go

```diff
@@ -58,3 +59,12 @@ func getClusterLocation(region, zone string) string {
 	}
 	return region
 }
+
+func getZoneFromLoc(location string) string {
```

Comment:
Unit test, please. :)

Reply:
Also `zoneFromLoc` would be a more Go-style name.

Reply:
done